src/report-python/__init__.py

```diff
@@ -278,3 +279,24 @@ def report(pd, io):
         return result
 
     result = report_problem(pd)
+
+def run_event_on_problem_dir(dump_dir_name, event_name, log_err=False):
+    """Helper function handling a single event
+
+    Keyword arguments:
+    envet_name -- a name of handled event
+    problem_dir -- a path to problem directory
+
+    Returns True if the handled event was successfully executed; otherwise
+    returns False.
+
+    """
+    state = run_event_state()
+    ret = state.run_event_on_dir_name(dump_dir_name, event_name)
+
+    if state.children_count == 0 and ret == 0:
```

Comment:
Maybe we should raise a RuntimeError exception here. Or better we should introduce a new exception type. Or even better the function could return a tuple.